pkg/tfgen/generate_schema.go

```diff
@@ -338,6 +338,12 @@ func (g *schemaGenerator) genPackageSpec(pack *pkg) (pschema.PackageSpec, error)
 		})
 	}
 
+	if javai := g.info.Java; javai != nil {
+		spec.Language["java"] = rawMessage(map[string]interface{}{
```

Comment:
So we have this type to model the contents of `.language.java`: https://github.com/pulumi/pulumi-java/blob/main/pkg/codegen/java/importer.go#L32 - it's a bit hidden but must be made more prominent. Is there a way to reuse the type or not worth it? Can do later.